Eatery Blue/main.swift

```diff
@@ -74,8 +74,6 @@ extension Networking {
 
     static var `default` = Networking(fetchUrl: URL(string: "https://eatery-dev.cornellappdev.com/eatery/")!)
```

Comment:
I know dev period is over, but write in the handoff docs to hide these secrets.

Reply:
Checkout Jayson/Secret branch to see if I did it right so i can push the pr I did that weeks ago just haven't asked

Reply:
I'll take a look after this PR is pushed.